--- adig.c
+++ adig.c
@@ -30,9 +30,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_STRINGS_H
 #  include <strings.h>

--- ares_config.h.cmake
+++ ares_config.h.cmake
@@ -51,9 +51,6 @@
 /* Define to 1 if you have the <arpa/inet.h> header file. */
 #cmakedefine HAVE_ARPA_INET_H
 
-/* Define to 1 if you have the <arpa/nameser_compat.h> header file. */
-#cmakedefine HAVE_ARPA_NAMESER_COMPAT_H
-
 /* Define to 1 if you have the <arpa/nameser.h> header file. */
 #cmakedefine HAVE_ARPA_NAMESER_H
 

--- ares_create_query.c
+++ ares_create_query.c
@@ -24,9 +24,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- ares_expand_name.c
+++ ares_expand_name.c
@@ -24,9 +24,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_nowarn.h"

--- ares_gethostbyaddr.c
+++ ares_gethostbyaddr.c
@@ -29,9 +29,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_inet_net_pton.h"

--- ares_gethostbyname.c
+++ ares_gethostbyname.c
@@ -30,9 +30,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_STRINGS_H
 #include <strings.h>

--- ares_getnameinfo.c
+++ ares_getnameinfo.c
@@ -36,9 +36,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_NET_IF_H
 #include <net/if.h>

--- ares_init.c
+++ ares_init.c
@@ -38,9 +38,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #if defined(ANDROID) || defined(__ANDROID__)
 #include <sys/system_properties.h>

--- ares_parse_a_reply.c
+++ ares_parse_a_reply.c
@@ -30,9 +30,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_STRINGS_H
 #  include <strings.h>

--- ares_parse_aaaa_reply.c
+++ ares_parse_aaaa_reply.c
@@ -31,9 +31,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_STRINGS_H
 #  include <strings.h>

--- ares_parse_mx_reply.c
+++ ares_parse_mx_reply.c
@@ -31,9 +31,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- ares_parse_naptr_reply.c
+++ ares_parse_naptr_reply.c
@@ -31,9 +31,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- ares_parse_ns_reply.c
+++ ares_parse_ns_reply.c
@@ -34,9 +34,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- ares_parse_ptr_reply.c
+++ ares_parse_ptr_reply.c
@@ -27,9 +27,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_STRINGS_H
 #  include <strings.h>

--- ares_parse_soa_reply.c
+++ ares_parse_soa_reply.c
@@ -31,9 +31,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- ares_parse_srv_reply.c
+++ ares_parse_srv_reply.c
@@ -31,9 +31,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- ares_parse_txt_reply.c
+++ ares_parse_txt_reply.c
@@ -31,9 +31,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_STRINGS_H
 #  include <strings.h>

--- ares_process.c
+++ ares_process.c
@@ -37,9 +37,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #ifdef HAVE_STRINGS_H
 #  include <strings.h>

--- ares_query.c
+++ ares_query.c
@@ -24,9 +24,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- ares_send.c
+++ ares_send.c
@@ -24,9 +24,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_dns.h"

--- inet_net_pton.c
+++ inet_net_pton.c
@@ -29,9 +29,6 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
 
 #include "ares.h"
 #include "ares_ipv6.h"

--- inet_ntop.c
+++ inet_ntop.c
@@ -28,9 +28,7 @@
 #else
 #  include "nameser.h"
 #endif
-#ifdef HAVE_ARPA_NAMESER_COMPAT_H
-#  include <arpa/nameser_compat.h>
-#endif
+
 
 #include "ares.h"
 #include "ares_ipv6.h"

